### PR TITLE
Add permissions advice to DropBox page

### PR DESCRIPTION
**Pull request by ximenesuk**
This PR advises sysadmins on how best to handle permissions changes of DropBox directories. The restart sequence in this PR needs testing. With DropBox running (confirm via diagnostics) disable and then stop both servers. Confirm via diagnostics. Then restart both servers in the order given. Both should be active according to diagnostics. Confirm that the MonitorServer log ends with something like,
```
...
2015-05-26 18:51:20,200 INFO  [                fsserver.fsMonitorServer] (Dummy-3   ) Monitor id = d07d34dc-03cf-11e5-8d75-3c075416fb46 created. Proxy: DropBox.default -t -e 1.1 @ DropBox.omerofs.DropBox
2015-05-26 18:51:20,201 INFO  [                fsserver.fsMonitorServer] (Dummy-5   ) Monitor id = d07d34dc-03cf-11e5-8d75-3c075416fb46 started
```

**Note**: If the log ends with just,
```
...
2015-05-26 18:50:59,357 INFO  [                  fsserver.MonitorServer] (MainThread) Started OMERO.fs MonitorServer
```
then the server has failed to start fully.

See: https://github.com/openmicroscopy/openmicroscopy/pull/3842#issuecomment-105436794

--no-rebase

### CODE REVIEW

**Comment by ximenesuk** [comment]
@joshmoore do you ant to check that the wording is reasonable and the command sequence is correct?

File: omero/sysadmins/dropbox.txt [hunk 1]
```diff
@@ -90,6 +90,31 @@ a given DropBox.
...
+
+    $ bin/omero admin server disable DropBox
+    $ bin/omero admin server stop DropBox
+    $ bin/omero admin server disable MonitorServer
+    $ bin/omero admin server stop MonitorServer
+
+    # make any directory changes
+
+    $ bin/omero admin server enable DropBox
+    $ bin/omero admin server enable MonitorServer
```
Comment (joshmoore) [inline omero/sysadmins/dropbox.txt]: In testing, I'm pretty sure I did these the other way 'round.

Comment (ximenesuk) [inline omero/sysadmins/dropbox.txt]: Hmm, okay, I'll test locally but I think you may be right.

**Comment by ximenesuk** [comment]
Thanks @joshmoore the other way around for the restart definitely works, though you do need `ice` in there as well!

**Comment by hflynn** [comment]
This looks fine on staging - good to merge @joshmoore ?

**Comment by joshmoore** [comment]
If we're going to have a merge build tomorrow AM anyway, can I suggest this gets another round of testing?

**Comment by hflynn** [comment]
Ok, I'll await a good to merge message tomorrow then.

File: omero/sysadmins/dropbox.txt [hunk 2]
```diff
@@ -90,6 +90,31 @@ a given DropBox.
         being imported depending on the operating system and how the image
         was modified.
 
+Permissions
+-----------
+
+Changing the permissions of a directory within DropBox may result in duplicate
+imports as a newly readable directory appears identical to a new directory. If
+directories need to be modfied it is recommended that the DropBox system is
```
Comment (mtbc) [inline omero/sysadmins/dropbox.txt]: modified

**Comment by joshmoore** [comment]
Beyond @mtbc's point, this looks good. The only thing I'm not sure of is to what extent it's clear enough how important this is. At least for 5.1.2 it would be *very* annoying if someone `chmod -R`'d their folder. Do we need to go further in warning?

NB1: I tested via this:
```
$ cat start.ice
admin ice server enable MonitorServer
admin ice server enable DropBox
$ bin/omero load < start.ice
```
which was quite nice.

NB2: in the DropBox log I saw:
```
2015-05-27 08:59:06,458 ERROR [                        fsclient.DropBox] (MainThread) Quitting
```
which seems to be a fluke that can periodically happen. (RFE).

**Comment by dominikl** [comment]
Tested with a 5.1.0 virtualbox image (updated with the 5.1.2-rc build of course), the instructions work like described.

**Comment by ximenesuk** [comment]
@joshmoore Does this extra note help. It slightly repeats the first note in that box but it does add emphasis and the next subsection is Permissions.

**Comment by joshmoore** [comment]
Works for me. Better safe than sorry, but will (*again*) be quiet if @hflynn wishes as much. :smile:

**Comment by hflynn** [comment]
Just rerunning the merge build but looks good

**Pull request merged**